packages/react/.storybook/static/assets/images/icons/burger.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is this need. This should come from `@oxygen-ui/react-icons`. Check other icons as well.

Reply:
Fixed in latest commits